### PR TITLE
Add & cleanup data in tables overview

### PR DESCRIPTION
**Pull request by ariser**
This PR introduces 2 changes:
1. Reformat & group table information in overview for better readability (4th commit)
2. Display new info about table read-only replicas in a new section of overview (5th commit)

the first 3 commits are technical updates to support the 4th one

### CODE REVIEW

File: src/containers/Tenant/Schema/SchemaInfoViewer/SchemaInfoViewer.scss [hunk 1]
```diff
@@ -1,6 +1,21 @@
 .schema-info-viewer {
     overflow: auto;
 
+    &__row {
+        display: flex;
+        flex-wrap: wrap;
+        justify-content: flex-start;
+        align-items: flex-start;
+        gap: 50px;
```
Comment (Raubzeug) [inline src/containers/Tenant/Schema/SchemaInfoViewer/SchemaInfoViewer.scss:9]: Flex gap is not supported id 14 Safari

**Review by Raubzeug — CHANGES_REQUESTED** [review]

File: src/containers/Tenant/Schema/SchemaInfoViewer/SchemaInfoViewer.scss (hunk 1 above)
Comment (ariser) [reply src/containers/Tenant/Schema/SchemaInfoViewer/SchemaInfoViewer.scss:9]: And still it is used all over the project. We probably should address that, if we target safari 14.
Are you suggesting we don't use it here?

**Review by Raubzeug — DISMISSED** [review]

File: src/services/api.js [hunk 2]
```diff
@@ -83,7 +83,6 @@ export class YdbEmbeddedAPI extends AxiosWrapper {
                 path,
                 enums: true,
                 backup: false,
-                partition_config: false,
```
Comment (ariser) [inline src/services/api.js:86]: partition data didn't load, fixed

File: src/components/InfoViewer/InfoViewer.scss [hunk 3]
```diff
@@ -27,11 +27,10 @@
 
     &__label {
         display: flex;
-        flex: 1 1 auto;
+        flex: 0 1 auto;
         align-items: baseline;
 
         min-width: 200px;
-        max-width: 200px;
```
Comment (ariser) [inline src/components/InfoViewer/InfoViewer.scss:34]: to allow longer labels widen the block, like this

**Review by Raubzeug — APPROVED** [review]

**Pull request merged**